source/DSCResources/OfficeOnlineServerSetup/OfficeOnlineServerSetup.schema.psm1

```
@@ -35,9 +35,9 @@ configuration OfficeOnlineServerSetup
         [string]
         $Path,
 
-        [Parameter()]
+        [Parameter(Mandatory = $true)]
```

PSScriptAnalyzer (1.24.0)

[error] Unexpected attribute 'Parameter'.

(UnexpectedAttribute)